pkg/client/client.go

```diff
@@ -1004,9 +1017,35 @@ func monitorKeepTerminatedEvent(
 					members,
 				)
 
-				tss.BroadcastRecoveryAddress(
+				if err != nil {
+					logger.Errorf(
+						"failed to retrieve member ids on keep [%s] termination: [%v]",
+						keep.ID().Hex(),
+						err,
+					)
+					return
+				}
+
+				var beneficiaryAddress string
+				_, err = btcutil.DecodeAddress(bitcoinConfig.BeneficiaryAddress, &chaincfg.TestNet3Params)
```

Comment:
We should pull `&chaincfg.TestNet3Params` to a parameter. Thoughts on defining it under `bitcoinConfig`?

Reply:
The default should be mainnet.